modules/utxo-lib/src/bitgo/UtxoPsbt.ts

```diff
@@ -218,10 +218,18 @@ export class UtxoPsbt<Tx extends UtxoTransaction<bigint>> extends Psbt {
     return results.reduce((final, res) => res && final, true);
   }
 
-  validateTaprootSignaturesOfInput(inputIndex: number): boolean {
+  validateTaprootSignaturesOfInput(inputIndex: number, pubkey?: Buffer): boolean {
     const input = this.data.inputs[inputIndex];
-    const mySigs = (input || {}).tapScriptSig;
-    if (!input || !mySigs || mySigs.length < 1) throw new Error('No signatures to validate');
+    const tapSigs = (input || {}).tapScriptSig;
+    if (!input || !tapSigs || tapSigs.length < 1) throw new Error('No signatures to validate');
```

Comment:
Nit: For readability and consistency in the repo.
```suggestion
    if (!input || !tapSigs || tapSigs.length < 1) {
      throw new Error('No signatures to validate');
    }
```